python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -619,7 +619,11 @@ def assembleMetadata(self, coaddExposure, tempExpRefList, weightList):
             self.inputRecorder.addVisitToCoadd(coaddInputs, tempExp, weight)
         coaddInputs.visits.sort()
         if self.config.doPsfMatch:
-            psf = self.config.modelPsf.apply()
+            # set PSF to modelPsf with maximum dimensions
```

Comment:
Both would be great!  If the commit message is descriptive enough, the inline can be kept as brief as possible to provide the basic reasoning.